File: data/src/main/java/com/everis/data/model/IntroMessageEntityDataMapper.java
package com.everis.data.model;

import com.everis.domain.model.IntroMessage;

/**
 * Created by everis on 26/04/18.
 */

public class IntroMessageEntityDataMapper {

    public static IntroMessage transform(IntroMessageEntity introMessageEntity){
        IntroMessage introMessage = new IntroMessage();
        introMessage.setDisplayMessage(introMessageEntity.getTitle() + introMessageEntity.getMessage());
        return introMessage;
    }

}

File: data/src/main/java/com/everis/data/model/IntroMessageEntityDataMapper.kt
package com.everis.data.model

import com.everis.domain.model.IntroMessage

object IntroMessageEntityDataMapper {

    fun transform(introMessageEntity: IntroMessageEntity): IntroMessage {
        val introMessage = IntroMessage()
        introMessage.displayMessage = introMessageEntity.title!! + introMessageEntity.message!!
        return introMessage
    }

}
